tests/acceptance/features/webUICreateFilesFolders/createFolders.feature

```diff
@@ -5,6 +5,7 @@ Feature: create folders
 
   Background:
     Given user "Alice" has been created with default attributes and without skeleton files
+    And user "Brian" has been created with default attributes and without skeleton files
```

Comment:
This can be removed from here. It is only needed in one scenario, so it the step can go just in that scenario.